### PR TITLE
chore(eng-1367): Update var values in samples, kittehub, and demos_internal

### PR DESCRIPTION
**Pull request by pashafateev**
This pull request is related to the following PRs:
- [demos_internal/pull/18](https://github.com/autokitteh/demos_internal/pull/18)
- [samples/pull/112](https://github.com/autokitteh/samples/pull/112)

### CODE REVIEW

File: github_copilot/autokitteh.yaml [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
 
 project:
   name: github_copilot
   vars:
     - name: GITHUB_ORG
-      value: "yourorg"
+      value: 
     - name: IDLE_USAGE_THRESHOLD
-      value: "72h"
+      value:
```
Comment (daabr) [inline github_copilot/autokitteh.yaml:21]: The explanation you added above is good, and this is indeed a required var, but having a reasonable default value like 72h seems fine to me in the specific case of `IDLE_USAGE_THRESHOLD`.

File: google_forms_to_jira/autokitteh.yaml [hunk 2]
```diff
@@ -10,12 +16,12 @@ version: v1
...
-    - name: FORM_ID
-      value: <Google Form ID> # TODO: Replace with Google Form ID
+    - name: GOOGLE_FORM_ID
+      value: 
     - name: JIRA_PROJECT_KEY
-      value: <Jira Project Key> # TODO: Replace with Jira Project Key
+      value: 
     - name: POLL_INTERVAL
-      value: 10
+      value:
```
Comment (daabr) [inline google_forms_to_jira/autokitteh.yaml:24]: Similar to the Copilot sample - `POLL_INTERVAL` is fine as a required var with a description at the top of the file, but a reasonable default is still useful in this specific case.

File: jira_assignee_from_calendar/autokitteh.yaml [hunk 3]
```diff
@@ -16,7 +16,7 @@ project:
   name: jira_assignee_from_calendar
   vars:
     - name: SHARED_CALENDAR_ID
-      value: primary
+      value:
```
Comment (daabr) [inline jira_assignee_from_calendar/autokitteh.yaml:19]: If "primary" is the API's default, I'd keep it as a default here too.

**Review by daabr — APPROVED** [review]
Approved with a few minor requests. The added documentation headers are useful!

File: aws_health_to_slack/autokitteh.yaml [hunk 4]
```diff
@@ -14,7 +14,7 @@ project:
   name: aws_health_to_slack
   vars:
     - name: GOOGLE_SHEET_URL
-      value: https://docs.google.com/spreadsheets/d/1PalmLwSZOPW9k668_jU-wFI5xCj88a4mDfNUtJAupMQ/
```
[the remark this answers is not in the document]
Comment (pashafateev) [reply aws_health_to_slack/autokitteh.yaml]: done

File: github_copilot/autokitteh.yaml (hunk 1 above)
Comment (pashafateev) [reply github_copilot/autokitteh.yaml]: Done

File: google_forms_to_jira/autokitteh.yaml (hunk 2 above)
Comment (pashafateev) [reply google_forms_to_jira/autokitteh.yaml]: Done

File: jira_assignee_from_calendar/autokitteh.yaml (hunk 3 above)
Comment (pashafateev) [reply jira_assignee_from_calendar/autokitteh.yaml]: Done

**Pull request merged**